src/Security/UserProvider/CustomerObjectUserProvider.php

```diff
@@ -82,7 +72,7 @@ public function refreshUser(UserInterface $user)
             throw new UnsupportedUserException();
         }
 
-        return $this->customerProvider->getById($user->getId(), true);
+        return $this->customerProvider->getById($user->getId(), ['force' => true]);
```

Comment:
This change conflicts with this change https://github.com/pimcore/customer-data-framework/pull/350/files

Side note: The draft branch seems does not have the latest commits, needs to be rebased.